tests/test_end_to_end.py

```diff
@@ -256,6 +261,8 @@ def test_simple_page_to_markdown():
     assert "Page content" in document_as_markdown
 
 
+
+
 def test_builtin_plugins(tmp_path):
```

Comment:
I was expecting that you would update the plugin end to end tests (i.e., this one where the comment is). See the issue for more details and let me know if it's not clear.

Reply:
Per the issue, I was hoping you'd "Update the existing plugin end-to-end test". I.e., you would update the `test_builtin_plugins`. This end to end test already loads all of the other builtin plugins and tests that they work (more or less). You can just add your new plugin and update the associated notion page to test it.

I don't think you should need to add any new end to end tests (i.e.,  you can probably remove `test_links_to_pages`) or modify the `test_all_blocks_page_to_markdown` test (that test is meant to test how things work without any plugins.